src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -300,6 +300,14 @@ def _validate_list(field_content: list, is_tensor_field: bool) -> None:
 
         return
 
+    @classmethod
+    def validate_multimodal_field_name(cls, multimodal_field_name:str) -> None:
+        cls.validate_field_name(multimodal_field_name)
+        # This error will never be raised because _id can't be a tensor_fields
+        # TODO check if there are other validations needed for multimodal_field_name
+        if multimodal_field_name == "_id":
```

Comment:
What is special about multimodal, since we're not doing this validation for other fields?